test/Atmos/EDMF/edmf_kernels.jl

```diff
@@ -372,6 +379,61 @@ function primitive_to_prognostic!(
 
 end
 
+function compute_filter_argument!(
+    target::SpecificEDMFFilter,
+    filter_state::Vars,
+    state::Vars,
+    aux::Vars,
+)
+    gm = state
+    up = state.turbconv.updraft
+    en = state.turbconv.environment
+    fs_up = filter_state.turbconv.updraft
+    fs_en = filter_state.turbconv.environment
+    ρ_inv = 1 / gm.ρ
+    N_up = n_updrafts(turbconv_model(target.atmos))
+
+    @unroll_map(N_up) do i
+        target.ρa && (fs_up[i].a = up[i].ρa * ρ_inv)
```

Comment:
I am not familiar with `&&` in Julia unroll maps, what is this doing?

Reply:
`unroll_map` wraps `i` in `Val` and manually unrolls the call to `map` ([here](https://github.com/CliMA/ClimateMachine.jl/blob/master/src/Utilities/VariableTemplates/VariableTemplates.jl#L369-L389)). This was some magic @simonbyrne wrote.

The syntax
```julia
foo() && bar()
```
is Julia short hand for
```julia
if foo()
bar()
end
```

Reply:
I get it now, pretty neat!